frontend/src/components/UpdateModal.tsx

```diff
@@ -34,7 +34,7 @@ export const UpdateModal: FC = () => {
               A new version of this tool is available.
               <Box>
                 Very soon this will let you self-update. Until this time, you
-                can download the latest version from the github repo.
+                can download the latest version from the <a href="https://github.com/olup/lunii-admin/releases/latest">github repo</a>.
```

Comment:
is this file still used ?

Reply:
Don't know, but it speaks about new version, so I thought it was interesting to give the link.

Reply:
So I have change this part according to your comments for the file [frontend/src/components/AboutModal.tsx](https://github.com/olup/lunii-admin/pull/4/files/49dfee7acd6d3ef719030709c5ff0c15bf4ef6d5#diff-bfea5e4ebc7627af16d65ebccf587edad9127fc3c250dbd3eea624005775dfe4)
Outdated

But maybe you prefer to delete this file?